rspace/src/main/scala/coop/rchain/rspace/history/InMemoryTrieStore.scala

```diff
@@ -41,10 +41,21 @@ class InMemoryTrieStore[K, V]
   private[this] val refine        = Map("path" -> "inmemTrie")
   private[this] val entriesGauge  = Kamon.gauge(MetricsSource + ".entries").refine(refine)
 
-  override private[rspace] def updateGauges(): Unit =
-    withTxn(createTxnRead())(_.readState { state =>
-      entriesGauge.set(state._dbTrie.size.toLong)
-    })
+  private[rspace] def updateGauges(): Unit = {
```

Comment:
This method looks a lot like it's twin brother from the `InMemoryStore`. It would make sense to factor out the common parts